src/page-racial-disparities/PageRacialDisparities.js

```diff
@@ -254,10 +254,11 @@ export default function PageRacialDisparities() {
         times.
       </p>
       <BodySizeP>
-        Due to a very small number of people with a race marked as Asian or
-        Native Hawaiian Pacific Islander in the North Dakota DOCR population
-        these racial groups have been included in the &ldquo;other&rdquo;
-        category.
+        Use the dropdown control on the right to investigate how a specific race
+        is affected by each part of the system. Due to a very small number of
+        people identified as Asian and Native Hawaiian or other Pacific Islander
```

Comment:
Is this the copy we want? "or other Pacific Islander"? Should it not just be "or Pacific Islander"?

Reply:
The official name for this racial group is "Native Hawaiian or Other Pacific Islander". I think the "other" word is there because Hawaii is technically a Pacific Island. Thoughts? Should we capitalize the "Other"?

Reply:
Yeah I think capitalization would help here, personally.